cpg_pipes/providers/cpg/inputs.py

```diff
@@ -241,3 +238,21 @@ def populate_pedigree(self, cohort: Cohort) -> None:
                 f'{dataset.name}: found pedigree info for {len(samples_with_ped)} '
                 f'samples out of {len(dataset.get_samples())}'
             )
+
+    def requested_dataset_names(self) -> list[str] | None:
+        """
+        List of dataset names to process. Provides defaults for the
+        populate_cohort(datsets=[]) parameter.
+        """
+        # Parsing dataset names from the analysis-runner Seqr stack:
+        from urllib import request
+
+        seqr_stack_url = (
+            'https://raw.githubusercontent.com/populationgenomics/analysis-runner/main'
+            '/stack/Pulumi.seqr.yaml'
+        )
```

Comment:
@illusional Could we get this from the SM server instead? That might be a bit cleaner, as our deployment config isn't really meant to be consumed by pipelines.

Reply:
This would be a good thing to address part of https://github.com/populationgenomics/sample-metadata/issues/220, we currently do not have a mechanism for determining this.